hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/TagType.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model;
+
+/**
+ * The type of Tag being stored or fethed.
+ *
+ * @author jay shaughnessy
+ * @author lucas ponce
+ */
+public enum TagType {
+    TRIGGER, ALERT, EVENT
```

Comment:
Perhaps this enum is part of the implementation instead of the model ?
From a pure API angle, tags are part of a component, isn't it ?